README.md

```diff
@@ -12,7 +12,7 @@
 
 Sequelize is a promise-based [Node.js](https://nodejs.org/en/about/) [ORM tool](https://en.wikipedia.org/wiki/Object-relational_mapping) for [Postgres](https://en.wikipedia.org/wiki/PostgreSQL), [MySQL](https://en.wikipedia.org/wiki/MySQL), [MariaDB](https://en.wikipedia.org/wiki/MariaDB), [SQLite](https://en.wikipedia.org/wiki/SQLite), [DB2](https://en.wikipedia.org/wiki/IBM_Db2_Family) and [Microsoft SQL Server](https://en.wikipedia.org/wiki/Microsoft_SQL_Server). It features solid transaction support, relations, eager and lazy loading, read replication and more.
 
-Sequelize follows [Semantic Versioning](http://semver.org) and supports Node v10 and above.
+Sequelize follows [Semantic Versioning](http://semver.org) and supports Node v12 and above.
```

Comment:
We should update this to a longer thing. So we will go to ^12.22.0 || ^14.17.0 || >=16.0.0 because of the support of ES modules. Do you happen to know how other projects tackle this?

Reply:
Maybe we could say, something like "follow SEmVer and supports all LTS versions of Node (12.xx, 14.xx, 16.xx)?